index.html

```diff
@@ -36,6 +36,196 @@ <h1>
             <li><a href="#"><img src="./assets/victory.svg" alt="victory"></a></li>
         </ul>
     </section>
+
+    <section id="works">
+        <div class="project">
+            <div>
+                <img src="./assets/SnapshootPortfolio.svg" alt="">
+            </div>
+
+            <div class="projectDescription">
+                <div class="projectTitle">
+                    <span class="heading">Tonic</span>
```

Comment:
- [ ] Great job removing the multiple h1 tags. But note that this is still a heading and it is best to use a semantic element. Kindly use heading tags between `h2 and h5` for all heading/titles.